### DISCUSSION TITLE
Please add dark mode in documentation page

### DISCUSSION THREAD

**Discussion by harshilsharma63** [tailwindlabs/tailwindcss#3265] [Ideas]
Please add dark mode in documentation page

**Comment by tomsdob** [comment]
https://tailwindcss.com/docs/dark-mode - isn't this already the dark mode documentation page? 🤔

**Comment by harshilsharma63** [comment]
@tomsdob let me re-phrase - please implement a dark mode for Tailwind documentation site :P

Something like this -
<img width="1446" alt="Screenshot 2021-01-04 at 9 08 59 PM" src="https://user-images.githubusercontent.com/18575143/103551975-3046a680-4ed1-11eb-8d5a-e5ac28ba622e.png">

**Comment by simonswiss** [comment]
Hey 👋

This would be a consequent amount of work to do properly (with all our examples etc), and we're a very small team with a lot of ambition to ship new things 😅

This has been discussed a few times, but I doubt we'll be able to prioritize it, at least in the foreseeable future.

If we were to do it, we'd want to do it with a lot of attention to details (and not just invert the shades of gray on everything), which in the context of our documentation website would be a large piece of work.

**Comment by Edgardon** [comment]
Thanks for sharing such great information, I highly appreciate your hard-working skills as the post you published have some great information which is quite beneficial for me, I hope you will post more like that in the future



[www.alaskasworld.com](https://alaskasworld.run/)
